# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/make_shlib_link.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makerules.orig	2002-10-12 11:52:02.000000000 +0200
+++ ./Makerules	2002-10-12 11:55:26.000000000 +0200
@@ -820,8 +820,7 @@
 endif
 ifndef make-shlib-link
 define make-shlib-link
-rm -f $@
-$(LN_S) $(<F) $@
+ln -sf $(<F) $@
 endef
 endif
 
--- ./Makefile.orig	2002-10-13 16:13:47.000000000 +0200
+++ ./Makefile	2002-10-13 16:16:26.000000000 +0200
@@ -110,7 +110,8 @@
 install: install-symbolic-link
 .PHONY: install-symbolic-link
 install-symbolic-link: subdir_install
-	$(symbolic-link-prog) $(symbolic-link-list)
+	while read from to ; do \
+		ln -sf $$from $$to ; done < $(symbolic-link-list)
 	rm -f $(symbolic-link-list)
 
 install:
